vault_core/src/vault_staker_withdrawal_ticket.rs

```diff
@@ -104,6 +103,13 @@ impl VaultStakerWithdrawalTicket {
         ])
     }
 
+    /// Returns the seeds for the PDA used for signing
+    pub fn signing_seeds(&self, vault: &Pubkey) -> Vec<Vec<u8>> {
+        let mut vault_seeds = Self::seeds(vault, &self.base);
+        vault_seeds.push(vec![self.bump]);
+        vault_seeds
+    }
+
```

Comment:
Use this function in conjunction with `Pubkey::create_program_address` in the `load` function below instead of using `self.find_program_address`

```
let expected_pubkey = Self::find_program_address(program_id, vault.key, &base).0;
// >
let expected_pubkey = Pubkey::create_program_address(seeds, program_id);
```

Reply:
Thank you!
I modified inside `load` function!